example_sns_init.yaml

```diff
@@ -133,19 +342,26 @@ Swap:
     # WARNING: Swap start_time works differently on mainnet and in testing.
     #
     # On mainnet:
-    # - Setting start_time to some value (e.g., 12:00 UTC) will allow the swap
+    # - Setting start_time to some value (e.g., 23:30 UTC) will allow the swap
     #   participants to be prepared for the swap in advance, e.g.,
     #   by obtaining ICPs that they would like to participate with.
     # - If start_time is not specified, the actual start time of the swap will
-    #   be chosen at random (allowing at least 24 hours, as described above).
+    #   be chosen at random (allowing at least 24 hours and less than 48 hours,
+    #   as described above).
     #
     # In testing:
     # - Setting start_time to some value works the same as explained above.
     # - If start_time is not specified, the swap will begin immediately after
     #   the CreateServiceNervousSystem proposal is executed. This facilitates
-    #   testing.
+    #   testing in an accelerated manner.
     #
-    # start_time: 12:00 UTC # noon
+    # start_time: 23:30 UTC
 
+    # The duration of the decentralization swap. When `start_time` is calculated
+    # during CreateServiceNervousSystem proposal execution, this `duration` will
+    # be added to that absolute time and set as the swap's deadline.
     duration: 7 days
+
+    # The amount of ICP that will be contributed to the decentralization swap by
+    # the Neurons' Fund if the CreateServiceNervousSystem proposal is adopted.
```

Comment:
It's the NNS Neurons' Fund but not sure if using NNS is established in this context (might make sense to explicitly distinguish between NNS and SNS here).